jbrowse/src/client/JBrowse/utils.ts

```diff
@@ -358,27 +359,59 @@ export async function fetchLuceneQuery(filters, sessionId, trackGUID, offset, pa
         sortReverse = false
     }
 
-    return Ajax.request({
-        url: ActionURL.buildURL('jbrowse', 'luceneQuery.api'),
-        method: 'GET',
-        success: async function(res){
-            let jsonRes = JSON.parse(res.response);
-            successCallback(jsonRes)
-        },
-        failure: function(res) {
-            console.error("There was an error: " + res.status + "\n Status Body: " + res.responseText + "\n Session ID:" + sessionId)
-            failureCallback("There was an error: status " + res.status)
-        },
-        params: {
-            "searchString": encoded,
-            "sessionId": sessionId,
-            "trackId": trackGUID,
-            "offset": offset,
-            "pageSize": pageSize,
-            "sortField": sortField ?? "genomicPosition",
-            "sortReverse": sortReverse 
-        },
+    const params = new URLSearchParams({
+        searchString: encoded,
+        sessionId,
+        trackId: trackGUID,
+        offset: offset,
+        pageSize: pageSize,
+        sortField: sortField ?? "genomicPosition",
+        sortReverse: sortReverse,
     });
+
+    try {
+        const response = await fetch(`/jbrowse/luceneQuery.api?${params.toString()}`);
```

Comment:
I think you should be using LABKEY.ActionURL.buildURL (i might have the case incorrect on the method name), rather than building this string yourself. Plus the URL depends on the containerPath and unless I'm missing something that's not accounted for here.